datasciencebr/rosie
# Simplify  import Rosie classifiers

**What is the purpose of this Pull Request?**
Make less repetitive importing a classifier. The result would be something like this:
**Before** 
`from rosie.chamber_of_deputies.classifiers.election_expenses_classifier import ElectionExpensesClassifier`
**After**
`from rosie.chamber_of_deputies.classifiers import ElectionExpensesClassifier`

The idea is just don't make necessary repeat the _classifier file_ and the _classifier class_ in sequence. At the end of this PR the idea is have **the same behaviour for all classifiers**.
**Before**
`from rosie.<app>.classifiers.<classifier_file> import <classifier class>`
**After**
`from rosie.<app>.classifiers import <classifier class>`

I changed a import on the [test_election_expenses_classifier](https://github.com/datasciencebr/rosie/pull/91/files#diff-3861458dcb40e69ca2d76d3d795045cfR6) to show how it would look like in practice

**What was done to achieve this purpose?**
I just used a trick on [`__init__`](https://github.com/datasciencebr/rosie/pull/91/files#diff-1b624ae1de251855d1affd5003964e5cR1) file.

**How to test if it really works?**
You can run Rosie tests and check if everything keep working well with `python rosie.py test`

**Who can help reviewing it?**
@cuducos @anaschwendler 

**TODO**
- [x] Changes for others classifiers
- [x] Changes everywhere that has a classifier import
